web/src/core/usecases/fileExplorer/state.ts

```diff
@@ -80,7 +80,7 @@ export const { reducer, actions } = createUsecaseActions({
                 directoryPath,
                 basename,
                 size,
-                uploadPercent: 0
+                uploadPercent: size === 0 ? 100 : 0
```

Comment:
Not sure here because maybe at first we want the file uploadPercent to be at `0`, and only then the `uploadProgress` method updates the content

Reply:
The action is dispatch before we start the request to s3. Even through we know the file size, the `uploadPercent` should be at 0.